policyengine_us/parameters/gov/states/va/tax/income/exemptions/personal.yaml

```diff
@@ -1,11 +1,11 @@
 description: Virginia provides an income tax exemption of this value for each person in the filing unit.
 values:
-  2022-01-01: 930
+  2021-01-01: 930
 
 metadata:
   period: year
   unit: currency-USD
-  label: Virginia personal exemption
+  label: Virginia personal exemption multiplier
```

Comment:
is this right?

Reply:
The exact phrase from the tax booklet is "Multiply the sum of exemptions claimed in the “You,” “Spouse” and “Dependents” boxes by $930."

Reply:
Should I name this to something else instead?